### PR TITLE
Jess account creation

### PR DESCRIPTION
**Pull request by mertzjl91**
Allows data to save to the DB and be sent back to front end if needed.

### CODE REVIEW

File: src/main/java/com/launchcode/AroundTownServer/models/User.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+
+import javax.persistence.Table;
+import javax.swing.*;
+
+@Entity
+@Table(name = "user")
```
Comment (lfallert) [inline src/main/java/com/launchcode/AroundTownServer/models/User.java:16]: unsure if this table annotation is needed, may not be necessary

File: src/main/java/com/launchcode/AroundTownServer/data/UserRepository.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package com.launchcode.AroundTownServer.data;
+
+import com.launchcode.AroundTownServer.models.User;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface UserRepository extends CrudRepository<User, Integer> {
+    User findByUsername(String username);
```
Comment (lfallert) [inline src/main/java/com/launchcode/AroundTownServer/data/UserRepository.java:9]: Unsure if line 9 is necessary for this feature, may b e able to delete and still have function

**Review by lfallert — CHANGES_REQUESTED** [review]
looks great, you did amazing with this! I think a couple lines may not be necessary for the feature, see comments

File: src/main/java/com/launchcode/AroundTownServer/models/User.java (hunk 1 above)
Comment (CNichole726) [reply src/main/java/com/launchcode/AroundTownServer/models/User.java:16]: it just names the table, so it shouldn't affect anything else

File: src/main/java/com/launchcode/AroundTownServer/data/UserRepository.java (hunk 2 above)
Comment (CNichole726) [reply src/main/java/com/launchcode/AroundTownServer/data/UserRepository.java:9]: I can't find a good resource right now to explain why, but it is necessary I'm pretty sure!

**Review by lfallert — APPROVED** [review]

**Review by CNichole726 — APPROVED** [review]
Looks good to me, and Laura and I discussed the questions she had

**Pull request merged**